src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -131,6 +131,8 @@ public string TokenTypeString
         public AssertionType AssertionType { get; set; }
 
         public CacheLevel CacheLevel { get; set; }
+
+        public string WamTelemetry { get; set; }
```

Comment:
Let's make this generic for all brokers, not just WAM.

Reply:
or just telemetry that would covered non-brokered flows when we enable runtime for non-brokered flow

Reply:
+1 for Iulian's comment. Long term, this will be used for non broker flows as well.